schedule/migration/autoyast_spvm_regression_test_offline_pre.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
is not spvm, we are using hmc for PowerVM.
Same comment, seems a generic installation. we using in YaST group @pvm at the end, but you could use for example: `autoyast_sle_spvm.yaml` or something like that, `offline` is only related with the linked test suite.

the other schedule below for auto-installation is exactly the same, we should just have one, and then directly chain the test suite that does rest of changes.